src/ERC20Mappings.ts

```diff
@@ -0,0 +1,9 @@
+import { Transfer } from './types/ERC20Detailed/ERC20Detailed';
+import { updateOpenInterest } from './utils/global-utils';
+
+export function handleERC20Transfer(event: Transfer): void {
+  let toAddress = event.params.to.toHexString();
+  let fromAddress = event.params.from.toHexString();
+  let amount = event.params.value;
+  updateOpenInterest(amount, toAddress, fromAddress);
+}
```

Comment:
It turns out mapping using the USDC contract will massively increase indexing time (realized this in discussions this evening), and because of this I think we should look to increment and decrement global TVL by having it updated from every action on the conditionalTokens contract and the FPMMs themselves. I will add other comments where necessary here.